getting_hired/README.md

```diff
@@ -1,20 +1,28 @@
-Web development is a lifelong journey of learning and growth. Continue that journey on a professional development team! You'll learn where to find jobs, how to do great interviews, and the best strategies to launch your career.
+# Getting Hired
 
-## The Outline
+This folder contains lesson markdown files that make up the <Getting Hired> course. This course exists in the [Full Stack Ruby On Rails](<https://www.theodinproject.com/paths/full-stack-ruby-on-rails>) and [Full Stack JavaScript](<https://www.theodinproject.com/paths/full-stack-javascript>) path on the Odin Project Website.
 
-- Preparing for Your Job Search
-  - How This Course Will Work - [lesson](introduction.md)
-  - Strategy - [lesson](strategy.md)
-  - It Starts with YOU - [lesson](starts_with_you.md)
-  - What Companies Want - [lesson](what_companies_want.md)
-  - What You Can Do To Prepare - [lesson](preparation.md)
-  - [Project: Building Your Personal Website](project_portfolio.md)
-- Applying to and Interviewing for Jobs
-  - Collecting Job Leads - [lesson](collect_leads.md)
-  - Qualifying Job Leads - [lesson](qualify_leads.md)
-  - [Project: Building Your Resume](project_resume.md)
-  - Applying for Web Development Jobs - [lesson](applying.md)
-  - Preparing to Interview and Interviewing - [lesson](preparing_to_interview.md)
-  - Handling a Job Offer - [lesson](handling_an_offer.md)
-  - Alternate Paths to a Web Development Job - [lesson](alternatives.md)
-  - Conclusion - [lesson](conclusion.md)
+## Course Outline
+
+The following list represents how the lessons are divided into sections and presented on the website.
+
+**Disclaimer:** Given the ever updating nature of the curriculum, the outline might be outdated. See the [Getting Hired course on the website](<https://www.theodinproject.com/paths/full-stack-ruby-on-rails/courses/getting-hired>)
+instead.
+
+### Preparing for Your Job Search
+
+1. [How This Course Will Work](preparing_for_job_search/how_this_course_will_work.md)
+2. [Strategy](preparing_for_job_search/strategy.md)
+3. [It Starts with YOU](preparing_for_job_search/starts_with_you.md)
+4. [What Companies Want](preparing_for_job_search/what_companies_want.md)
+5. [What You Can Do To Prepare](preparing_for_job_search/what_you_can_do_to_prepare.md)
+6. [**Project: Building Your Personal Website**](preparing_for_job_search/project_portfolio.md)
+### Applying to and Interviewing for Jobs
+
+1. [Collecting Job Leads](applying_and_interview/collect_leads.md)
+2. [Qualifying Job Leads](preparing_for_job_search/qualify_leads.md)
+3. [**Project: Building Your Resume**](preparing_for_job_search/project_resume.md)
+4. [Applying for Web Development Jobs](preparing_for_job_search/applying.md)
+5. [Preparing to Interview and Interviewing](<preparing_for_job_search/prearing_to_interview_and_interviewing.md)
+6. [Handling a Job Offer](preparing_for_job_search/handling_an_offer.md)
+7. [Conclusion](preparing_for_job_search/conclusion.md)
```

Comment:
Links are broken for all the lesson markdown files in the **Applying to and Interviewing for Jobs** section. These need to be fixed.